tests/jvm/src/test/resources/metacp.expect

```diff
@@ -1834,7 +1836,8 @@ flags.p.package.`y_=`(). => protected var method y_=(x$1: Int): Unit
   Unit => scala.Unit#
 flags.p.package.`y_=`().(x$1) => param x$1: Int
   Int => scala.Int#
-flags.p.package.m(). => macro m[TT]: Nothing
+flags.p.package.m(). => @macroImpl macro m[TT]: Nothing
+  macroImpl => scala.reflect.macros.internal.macroImpl#
   TT => flags.p.package.m().[TT]
   Nothing => scala.Nothing#
 flags.p.package.m().[TT] => typeparam TT
```

Comment:
It'd be great to have more tests. Can you create `Annotations.scala`in `semanticdb/integration/src/main/scala/example`, and then try annotating all kinds of symbols? For an exhaustive list of symbol kinds, please refer to https://github.com/scalameta/scalameta/blob/master/semanticdb/semanticdb3/semanticdb3.md#symbolinformation.

Reply:
It's not possible to annotate packages in scala https://github.com/scala/bug/issues/3600. If I try to do it from java, it crashes the compiler. Ibid for package objects. For Self-Types, I'm not sure if it's possible. I got all other cases covered.

Reply:
Nice job covering all the different kinds!